### DISCUSSION TITLE
Multi display: HD44780 + TM1637

### DISCUSSION THREAD

**Discussion by hrandib** [arendst/Tasmota#16950] [Q&A] [answered]
According to the documentation, there is no standard way to use these displays simultaneously. I see a workaround, implementing a complete TM1637 driver in Berry. Would it be the most straightforward way or I've missed some simpler solution?

**Comment by Jason2866** [comment, marked as answer]
Correct, Tasmota does not support more than one display.

**Reply by hrandib** [reply]
For all who are faced with the same problem, here is the solution: https://gist.github.com/hrandib/e813847f2683baf2169e0d6883317cb9
This TM1637 driver might be used in addition to any standard Tasmota-supported display.

**Reply by barbudor** [reply]
Well done
I think gist may have a limited life time
You may contribute your driver to Tasmota in the Berry/driver folder like those:
https://github.com/arendst/Tasmota/tree/development/tasmota/berry/drivers

**Reply by hrandib** [reply]
I'll prepare PR after polishing. Need some time for testing, it was a fast solution and my first experience with Berry. All my gists created are still available, the oldest is 4 years old.

**Reply by hrandib** [reply]
The PR has been created: https://github.com/arendst/Tasmota/pull/16982

**Reply by barbudor** [reply]
Thanks

**Reply by gemu2015** [reply]
@Jason2866 
> Correct, Tasmota does not support more than one display.

to clarify this:

LVGL and berry support only one display

displaytext supports up to 4 simultaneous displays
with the following requirement:
all drivers except the first one must be renderer based drivers.

so you may combine e.g. one TM1637 (none renderer based)
with one SD1306 (renderer based)
